assets/Styles/main.css

```diff
@@ -0,0 +1,904 @@
+@font-face {
+  font-family: Cocogoose;
+  src: url('/assets/fonts/Cocogoose_trial.woff');
+}
+
+@font-face {
+  font-family: Lato;
+  src: url('/assets/fonts/Lato-Regular.woff');
+}
+
+/* Custom properties */
+
+:root {
+  --primary-red: #ec5242;
+  --primary-white: #d3d3d3;
+  --primary-black: #272a31;
+  --transparency: rgba(62, 66, 73, 0.6);
+  --primary-ff: cocogoose;
+  --secondary-ff: lato;
+  --box-shadow: 0 -1px 6px 2px rgba(0, 0, 0, 0.25);
+  --background: #f5f5f5;
+}
+
+/* Core Styles */
+
+#hambMenu {
+  cursor: pointer;
+}
+
+.hidden {
+  display: none;
+}
+
+#menuMobile {
+  background-image: url("/assets/png/Header_mobile.png");
+  background-size: cover;
+  width: 100%;
+  height: 131px;
+  display: block;
+}
+
+.featCard img {
+  grid-column: 1/2;
+  grid-row: 1/3;
+}
+
+#menuMobile img {
+  padding: 1rem 0 0 2rem;
+}
+
+#menuDisplay {
+  background-color: var(--primary-black);
+  position: absolute;
+  width: 100%;
+  height: 100vh;
+  padding: 2rem;
+  flex-direction: column;
+}
+
+#menuDisplay a {
+  color: var(--primary-white);
+  text-decoration: none;
+  font-size: 3rem;
+  padding: 3rem 0 0 2rem;
+}
+
+#closeBtn {
+  float: right;
+  color: var(--primary-white);
+  padding-right: 1rem;
+  cursor: pointer;
+  font-family: var(--secondary-ff);
+  font-style: normal;
+}
+
+nav {
+  width: 100%;
+  display: none;
+}
+
+#socialBar {
+  display: none;
+}
+
+main {
+  width: 100%;
+  height: auto;
+}
+
+#heroSection {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 1rem 1rem 4rem 1rem;
+  background-image:
+    url("/assets/png/Ellipse-color-pattern.png"),
+    url("/assets/png/deco-0-icon.png"),
+    url("/assets/svg/x-gray-icon.svg"),
+    url("/assets/png/deco-dollar-icon.png");
+  background-repeat: no-repeat;
+  background-size: 20%, 5%, auto, auto;
+  background-position: 98% 39%, 13% 37%, 16% 98%, 89% 91%;
+}
+
+#heroSection img {
+  padding-bottom: 1rem;
+}
+
+#heroSection h1 {
+  color: var(--primary-red);
+  font-size: 2rem;
+  line-height: 90%;
+  font-family: var(--primary-ff);
+  font-weight: 700;
+  padding-bottom: 1rem;
+}
+
+#heroSection h3 {
+  color: var(--primary-red);
+  font-family: var(--secondary-ff);
+  font-weight: 300;
+}
+
+#heroSection h2 {
+  color: var(--primary-black);
+  font-family: var(--primary-ff);
+  font-weight: 800;
+}
+
+footer p {
+  align-self: center;
+  font-family: var(--secondary-ff);
+  font-size: 12px;
+  color: var(--primary-black);
+  font-weight: 300;
+}
+
+.featCard p {
+  grid-column: 2/3;
+  grid-row: 3/4;
+  font-size: 12px;
+}
+
+#heroSection p {
+  color: var(--primary-black);
+  font-size: 0.85rem;
+  font-family: var(--secondary-ff);
+}
+
+#sumContainer {
+  align-self: center;
+  width: 98%;
+  font-size: 1rem;
+  text-align: justify;
+  margin: 1rem 0;
+  padding: 1rem;
+  background-color: var(--background);
+  border: 1px solid white;
+}
+
+#mainProgram {
+  background-image: url(/assets/svg/mobile-background.svg);
```

Comment:
- [ ] **This background image is also not loaded in your live version. Please fix it**

![image](https://user-images.githubusercontent.com/59168430/125127321-f70a2000-e119-11eb-9868-7c1ce59df4c5.png)

Reply:
Fixed :heavy_check_mark: